ts/backend/src/app/features/controller/results.controller.mts

```diff
@@ -569,7 +574,127 @@ export class ResultsController extends Controller {
         }
       }),
     }
-    this.respond(res, result as json)
+    this.respond(res, [result])
+  }
+
+  /**
+   * @swagger
+   * /results/campaign-item/{benchmark_id}/tests/{test_id}:
+   *  get:
+   *    description: Get campaign item test leaderboard.
```

Comment:
Would this not be the same for competition setting: which submissions have which results at test level? -> Rename endpoint.